string-manipulation-3/src/is-palindromic.js

```diff
@@ -0,0 +1,9 @@
+/* exported isPalindromic */
+function isPalindromic(string) {
+  var stringSplit = string.split(' ').join('');
+  var reversedString = string.split('').reverse().join('').split(' ').join('');
```

Comment:
Maybe use `stringSplit` here instead of `string` since you already removed the spaces, no need to remove the spaces again